sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/ServiceVersion.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter
+{
+    /// <summary>
+    /// These values define the version of the Azure Monitor Ingestion Service.
+    /// </summary>
+    [SuppressMessage("Naming", "CA1707:Identifiers should not contain underscores", Justification = "This naming format is defined in the Azure SDK Design Guidelines")]
+    public enum ServiceVersion
```

Comment:
This class should be nested into client options see https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClientOptions.cs#L23

Reply:
oh okay, that's an easy change :)